### ISSUE TITLE
image normalization

### ISSUE THREAD

**Issue by norahik** [Neerajj9/CSRNet-keras#6]
```
im[:,:,0]=(im[:,:,0]-0.485)/0.229
im[:,:,1]=(im[:,:,1]-0.456)/0.224
im[:,:,2]=(im[:,:,2]-0.406)/0.225
```

why were these numbers specifically chosen (0.485 , 0.229 ,  0.456 , 0.224 , 0.406 , 0.225)?

**Comment by aditya-malte** [comment]
Hi norahik,
These (0.485, 0.229,etc) stand for the exact mean and std values of the ImageNet dataset that the VGG was pre-trained on.
Hence we normalised using these values.
Thanks.

**Comment by norahik** [comment]
thanks for your answer.
but can this be consistent even when using dataset other than ImageNet?

**Comment by aditya-malte** [comment]
@norahik yes
